src/screens/Ticketing/Tickets/TicketsScrollView.tsx

```diff
@@ -86,7 +86,11 @@ const TicketsScrollView: React.FC<Props> = ({
           />
         )}
         {!fareContracts?.length && !reservations?.length && (
-          <ThemeText style={styles.noTicketsText}>{noTicketsLabel}</ThemeText>
+          <MessageBox
+            containerStyle={styles.messageBox}
```

Comment:
I'm guessing this can be a normal use case here. We have horizontal padding but want to have padding under. The component isn't really working as intended when we have to add style to it, but also may be a problem with React Natives layout model not having the ability to do proper flex gaps.

Not necessary for this task, but I think we should reevaluate if we need to change the API of MessageBox to support margin bottom as a common case. Maybe with something like `margin: 'horizontal' | 'bottom' | 'bottom-horizontal';` etc.

Reply:
Yes, I think so too. But maybe not for this task at we would have to verify all the places it is being used. Don't want to increase the scope of this, so not touching this one.